RadixWallet/Features/TransactionReviewFeature/TransactionReview+Sections.swift

```diff
@@ -361,6 +422,29 @@ extension TransactionReview {
 		return .init(accounts: depositAccounts, enableCustomizeGuarantees: requiresGuarantees)
 	}
 
+	func extractValidators(for addresses: [EngineToolkit.Address]) async throws -> ValidatorsState? {
+		guard !addresses.isEmpty else { return nil }
+
+		let generalAddresses = try addresses.map { try $0.asSpecific() as Address }
```

Comment:
`let GENERAL = ... $0.asSPECIFIC`  sound like an oxymoron?

Reply:
It *is* correct as written, but I should use `.asGeneral` instead, I changed this around a few times and missed it

Reply:
Actually we didn't even have an option to convert directly from EngineToolkit.Address to the general address type, but I have added that now